orchagent/nexthopkey.h

```diff
@@ -15,9 +17,24 @@ struct NextHopKey
     string              alias;          // incoming interface alias
     uint32_t            vni;            // Encap VNI overlay nexthop
     MacAddress          mac_address;    // Overlay Nexthop MAC.
+    LabelStack          label_stack;    // MPLS label stack
```

Comment:
Suggest adding a next hop type, it can be modeled after SAI Nexthop object types. (IP tunnel, LSP tunnel)

Reply:
No apparent use case.

Reply:
@smaheshm I actually need this to properly program POP NHs now, so there is a use case.